core/common/src/main/java/alluxio/AbstractMasterClient.java

```diff
@@ -34,34 +34,24 @@ public abstract class AbstractMasterClient extends AbstractClient {
    * @param clientConf master client configuration
    */
   public AbstractMasterClient(MasterClientContext clientConf) {
-    super(clientConf, null);
+    super(clientConf);
     mMasterInquireClient = clientConf.getMasterInquireClient();
   }
 
   /**
    * Creates a new master client base.
    *
    * @param clientConf master client configuration
-   * @param address address to connect to
    * @param retryPolicySupplier retry policy to use
    */
-  public AbstractMasterClient(MasterClientContext clientConf, InetSocketAddress address,
-      Supplier<RetryPolicy> retryPolicySupplier) {
-    super(clientConf, address, retryPolicySupplier);
+  public AbstractMasterClient(MasterClientContext clientConf,
+                              Supplier<RetryPolicy> retryPolicySupplier) {
```

Comment:
nit: the indentation should be 4 extra spaces for broken lines

Reply:
Done